.github/CODEOWNERS

```
@@ -3,8 +3,11 @@
 # For syntax details see:
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-# the maintainers of the website
-* @voborgus @clcoffey
+# CODEOWNERS defaults to webiste-maintainers team
+* @InnerSourceCommons/website-maintainers
+
+# Learning Path
+content/learn/learning-path @InnerSourceCommons/learning-path
```

```suggestion
/content/learn/learning-path @InnerSourceCommons/learning-path
```

I would opt for specifying the absolute folder location i.e. starting at root (`/`)